tests/cypress/e2e/form/destination_config_fields/template.cy.js

```diff
@@ -0,0 +1,107 @@
+        cy.changeProfile('Super-Admin', true);
+
+        // Create form
+        cy.createFormWithAPI().as('form_id').visitFormTab('Form');
+
+        // Add a default question
+        cy.findByRole('button', {'name': "Add a new question"}).click();
+        cy.findByRole('button', {'name': 'Save'}).click();
+
+        // Create a ticket template
+        cy.get('@form_id').then((form_id) => {
+            const ticket_template_name = 'Test ticket template for the template configuration suite - ' + form_id;
+
+            cy.createWithAPI('TicketTemplate', {
+                'name': ticket_template_name,
+            }).as('ticket_template_id');
+
+            cy.get('@ticket_template_id').then((ticket_template_id) => {
+                cy.createWithAPI('TicketTemplateHiddenField', {
+                    'tickettemplates_id': ticket_template_id,
+                    'num': 12,
+                });
+            });
+        });
+
+        // Go to destination tab
+        cy.findByRole('tab', {'name': "Items to create"}).click();
+        cy.findByRole('button', {'name': "Add ticket"}).click();
+        cy.findByRole('alert').should('contain.text', 'Item successfully added');
+    });
+
+    it('can use all possibles configuration options', () => {
+        cy.findByRole('region', {'name': "Template configuration"}).as("config");
+        cy.get('@config').getDropdownByLabelText('Template').as("template_dropdown");
+
+        // Default value
+        cy.get('@template_dropdown').should(
+            'have.text',
+            'Default template'
+        );
+
+        // Make sure hidden dropdowns are not displayed
+        cy.get('@config').getDropdownByLabelText('Select a template...').should('not.exist');
+
+        cy.get('@form_id').then((form_id) => {
+            const ticket_template_name = 'Test ticket template for the template configuration suite - ' + form_id;
+
+            // Switch to "Specific template"
+            cy.get('@template_dropdown').selectDropdownValue('Specific template');
+            cy.get('@config').getDropdownByLabelText('Select a template...').as('specific_template_id_dropdown');
+            cy.get('@specific_template_id_dropdown').selectDropdownValue(ticket_template_name);
+
+            cy.findByRole('button', {'name': 'Update item'}).click();
+            cy.get('@template_dropdown').should('have.text', 'Specific template');
+            cy.get('@specific_template_id_dropdown').should('have.text', ticket_template_name);
+        });
+
+        // Go to preview
+        cy.findByRole('tab', {'name': "Form"}).click();
+        cy.findByRole('link', {'name': "Preview"})
+            .invoke('removeAttr', 'target') // Cypress can't handle tab changes
+            .click()
+        ;
+
+        cy.findByRole('button', {'name': 'Send form'}).click();
+        cy.findByRole('link', {'name': 'My test form'}).click();
+
+        // Check ticket values
+        cy.getDropdownByLabelText('Status').should('not.exist');
```

Comment:
Maybe this part could be in a separate `it` ?

If we compare with existing  tests, `can use all possibles configuration options` is used to set up the different config option and we generally have a separate test to show that the default config is working.